config/environment.js

```diff
@@ -148,16 +152,22 @@ module.exports = function (environment) {
       // when it is created
     },
 
-    MIN_PREFIXES_AVAILABLE:  minPrefixesAvailable,
+    MIN_PREFIXES_AVAILABLE: minPrefixesAvailable,
     SHOW_N_PREFIXES: showNPrefixes,
-    MAX_MINT_FUTURE_OFFSET:  maxMintFutureOffset
+    MAX_MINT_FUTURE_OFFSET: maxMintFutureOffset,
+    HANDLE_SERVER: ((typeof process.env.HANDLE_SERVER === 'undefined') || (process.env.HANDLE_SERVER == "")) ? 'https://handle.stage.datacite.org' : normalizeURL(process.env.HANDLE_SERVER)
   };
 
   if (fabricaDeployTarget === 'stage') {
     // add staging-specific settings here
     ENV.COOKIE_DOMAIN = '.stage.datacite.org';
   }
 
+  if (fabricaDeployTarget === 'test') {
```

Comment:
Does this match doi.test? i forget

If so shouldnt it default to handle.test.datacite.org? or are we gonna set that in the var?

Reply:
You are correct.  I will fix it.  It can also be handled in vercel through the env variables, but I will fix this.  Sorry about that.

Reply:
I am also looking at you can the a variable to the appropriate value for the environment in github actions and have it used by vercel so we can just set them in the github actions.